nautobot/extras/models/groups.py

```diff
@@ -248,11 +248,13 @@ def _map_filter_fields(self):
 
         # Filter out unwanted fields from the filterform
         for filterset_field_name, filterset_field in filterset_fields.items():
-
             # Skip filter fields that have methods defined. They are not reversible.
             if skip_method_filters and filterset_field.method is not None:
                 # Don't skip method fields that also have a "generate_query_" method
-                if hasattr(filterset, "generate_query_" + filterset_field.method):
+                query_attr = (
+                    filterset_field.method if isinstance(filterset_field.method, str) else filterset_field.method.name
```

Comment:
As many things can be cast to a string, I think you should reverse the check here and do `if callable(filterset_field.method)` instead.